### DISCUSSION TITLE
export reports doesn't work

### DISCUSSION THREAD

**Discussion by thejaqueta** [kanboard/kanboard#5529] [Q&A]
I want export general reports but doesn't work, I click to export but the button don't do nothing.

How can I repair this?

![Captura de tela 2024-08-07 141929](https://github.com/user-attachments/assets/14c84f48-c81d-4053-a5c8-24eb64f22341)

**Comment by petrik68** [comment]
Same problem here - I have installed a punch of plug-in's. One of those could cause the trouble: 

Google as authentication provider
Mailgun | Frédéric Guillot | 1.0.11 |
Calendar | Frédéric Guillot, Alfred Bühler | 1.5.0 
Gantt | Frédéric Guillot | 1.0.6 
TimeTrackingEditor | Thomas Stinner | 1.0.21 
Coverimage | BlueTeck + creecros | 1.2.14.1
FontSwitcher | aljawaid | 1.6.0 
KanboardCSS | aljawaid | 3.6.0 
ApplicationBranding | aljawaid | 4.0.0
ContentCleaner | aljawaid | 1.0.0 
***
Before i start to uninstall one-by-one those plug-in's does, anyone found if any of plugin's will cause the reporting function failure?
